packages/mobile/src/common/interfaces/global-styles/global-styles.ts

```diff
@@ -0,0 +1,38 @@
+export interface IFontStyles<T> {
```

Comment:
In that case generic has no sense as it will always be the same `TextStyle`. Same applies further
Also let's stick with type = {} notation just for the sake of consistency across the project

Reply:
corrected